test/metabase/driver/sql/query_processor_test_util.clj

```diff
@@ -2,83 +2,12 @@
   (:require
    [clojure.string :as str]
    [clojure.test :refer :all]
-   [honeysql.format :as hformat]
+   [metabase.db.query :as mdb.query]
    [metabase.driver :as driver]
    [metabase.driver.util :as driver.u]
    [metabase.query-processor :as qp]
    [metabase.util :as u]))
 
-;; [[install-pretty-formatters!]] -- tweaks the actual methods HoneySQL uses to generate SQL strings to add indentation
-
-(defonce ^:private ^clojure.lang.MultiFn orig-format-clause hformat/format-clause)
```

Comment:
Now that we have `mdb.query/format-sql`, we don't need to do a half-assed hack to try to get Honey SQL to spit out pretty SQL. It doesn't work for Honey SQL 2 anyway.